### DISCUSSION TITLE
Relação de tarefas e conhecimento (estagiário / jr / pleno / senior)

### DISCUSSION THREAD

**Discussion by theandersonn** [frontendbr/forum#331] [Discussão]
Passeando, encontrei o repositório "[CarreiraPHP](https://github.com/abraphp/CarreiraPHP)"  que achei bem interessante.

Das experiências que vcs tem aí nos seus trampos, qual seria o conhecimento e tarefas de cada cargo? Por exemplo numa agência onde as tarefas se pluralizam.

Olhei aqui no fórum e já rolou algumas discussões bem legais [aqui](https://github.com/frontendbr/forum/issues/252) e [aqui](https://github.com/frontendbr/forum/issues/92). 

Vamos montar um listinha? Acho que vai ajudar a galera se situar e saber por onde começar pra onde ir e o que focar nos estudos.

**Comment by woliveiras** [comment]
Eu fiz uma relação [aqui](https://github.com/woliveiras/front-end-career), @theandersonn :smile: 

Quem puder contribuir ganha um coraçãozinho (:heart:) meu!

**Comment by danilovaz** [comment]
Essa lista do @woliveiras é bem foda!

**Comment by hugoalmeidahh** [comment]
Dei uma lida na lista do @woliveiras , @danilovaz , isso é bem aquilo que te contem ontem, onde eu fui conversar com um SEO sobre uma oportunidade e o cara me questiou sobre o meu nível de conhecimento, no meu conhecimento em que eu me enquadrava, eu respondi Pleno, não só pelo conhecimento de linguagem mas como um todo, eu consigo me virar sozinho, ai ele me vira e diz que pela norma sindical Pleno é à partir de 6 anos de registro em carteira, e como eu só tinha 2 anos eu não seria Pleno.

**Comment by woliveiras** [comment]
Rapaz, norma sindical??? 

😱

**Comment by hugoalmeidahh** [comment]
Sim, nosso sindicato (SINDPD) tem umas portarias que ditam essas "Regras" de nivelamento.! eu tenho que parar um momento para ver melhor isso... pois fica muito engessado... beneficiando claramente a empresa ..!

**Comment by theandersonn** [comment]
@woliveiras Tinha esquecido a lista que vc fez, maneira mesmo. Se rolar sugestões aqui de tarefas a gente poderia add lá em alguma categoria, tinha pensado em algo bem direto, tipo o repositório de php que citei.

@hhalmeida Sabia desse lance do sindicato não, acho que depende do local tbm, porque se o cara consegue demonstrar que é capaz pra função está além anos trabalhados, gradução cursada e etc...

**Comment by danilovaz** [comment]
@theandersonn Eu sou CLT e também sou "regido" pelo SINDPD(Sindicato dos Trabalhadores em Processamento de Dados e Tecnologia da Informação). Doideira, não? Mas aqui na empresa não tem essa de 6 anos para ser pleno ahahhahaha

**Comment by hugoalmeidahh** [comment]
@danilovaz , acho que o @theandersonn se referiu lugar como empresa... essa para mim tbem foi a primeira vez que ouvi isso!...
Para mim isso é história para boi dormir, para não falar que vc é um profissional caro e o cara não quer pagar o que vc quer / merece!

**Comment by lucasmahle** [comment]
Temos que levar em consideração o contratante quando falamos em tempo. 
Sabemos que um programador preguiçoso não consegue necessariamente atingir um conhecimento de Pleno com 6 anos.
Já um programador genial (igual a todos desse fórum :stuck_out_tongue:) consegue atingir a "plenitude" em 1 anos ou menos, ai a variação é maior.
Mas acho ridículo dizer que só depois de 6 anos o cara seria pleno, então se o contratante considerar o tempo como aspecto de nivelamento, talvez ele não tenha bons critérios de contratação.

**Comment by ranosi** [comment]
Que doc irado esse heim @woliveiras . Vi que tem algumas infos sobre trabalhar fora também daí me lembrou a questão das certificações. Como funcionam na área? Seria legal acrescentar também alguma coisa sobre isso.

**Comment by woliveiras** [comment]
Valeu @ranosi!!!

Nunca vi certificações pra área, só cursos mesmo.

Manja alguma?

**Comment by bernardodiasc** [comment]
> talvez ele não tenha bons critérios de contratação.

com certeza, isso é quando alguem que nao manja nada tenta avaliar uma parada qualificada... se o cara vem com essa amadorisse pra cima de mim levanto e vo embora... kkkkkk

**Comment by ranosi** [comment]
@woliveiras ouvi alguma coisa sobre isso tempos atrás. Vou dar uma olhada com calma nessa questão. To começando a estudar as cosias agora, só 'junior' ainda perto de vocês aqui rs

**Comment by Lucasktrindade** [comment]
Acho que você tem que incluir nisso ai que empresas mandam um perfil para o RH, e como eles não entendem nada sobre, não olham os projetos que o cara fez, e olham apenas para tempo de experiência.

**Comment by pedrofelipe** [comment]
Quanto ao conhecimento, deixo minhas sugestões: [Júnior](https://github.com/Codeminer42/CarreiraRuby/blob/master/NivelJunior.md), [Pleno](https://github.com/Codeminer42/CarreiraRuby/blob/master/NivelPleno.md) e [Sênior](https://github.com/Codeminer42/CarreiraRuby/blob/master/NivelSenior.md)

👍
